cdap-cli/src/main/java/co/cask/cdap/cli/util/table/AltStyleTableRenderer.java

```diff
@@ -15,9 +15,11 @@
  */
```

Comment:
Don't think this should be part of this PR. Not sure why it's here though, since these changes are already in develop.